### ISSUE TITLE
Feature request: js_vars to use media query when function is style

### ISSUE THREAD

**Issue by carasmo** [themeum/kirki#1324]
Currently if I have the following:

```
//* h3 font size
Prefix_Kirki::add_field( 'prefix_theme', array(
	'type'        => 'slider',
	'settings'    => 'h3_font_size',
	'label'       => __( 'H3 Font Size', 'text-domain' ),
	'section'     => 'font_sizes',
    'transport'   => 'postMessage',
	'default'     => '30',
	'choices'     => array(
		'min'  => '10',
		'max'  => '120',
		'step' => '1',
	),	
	'priority'    => 10,
	'output'      => array(
		array(
			'element'     => 'h3, .h3',
			'property'    => 'font-size',
			'units'       => 'px',
			'media_query' => '@media (min-width: 1200px)',
		),
	),
	'js_vars'      => array(
		array(
			'element'     => 'h3, .h3',
			'property'    => 'font-size',
			'units'       => 'px',
			'function'    => 'style',
		),
	),
    'active_callback'    => array( 
        array(
            'setting'  => 'font_size_switch',
            'operator' => '==',
            'value'    => '1',
        ),
    ),	
) );
```

It works BUT the preview reflects all viewport widths. Rather than putting a verbose description on how when the user clicks the various viewport icons in the footer of the customizer panel and how it will not reflect on a live site, and so forth, I have to do the following in my customizer .js and I remove the js_vars on the control.

```
   //h3
    wp.customize( 'h3_font_size', function( value ) {

        value.bind( function( to ) {
            var a, b;

            a = '<style class="h3_font_size">@media (min-width: 1200px) {' +
                'h3, .h3 { font-size: ' + to + 'px; }' +
                '}</style>';


            b = $( '.h3_font_size' );
            if ( b.length ) {
                b.replaceWith( a );
            } else {
                $( 'head' ).append( a );
            }

        } );

    } ); //end
```

So some release, please consider adding the ability to use media queries in the js_vars too.

Thanks!

**Comment by aristath** [comment]
Closing as duplicate of #1184 
I've got this planned for v3.0 👍

**Comment by aristath** [comment]
Fixed in 3.0
